lib/stat_tracker.rb

```diff
@@ -1,97 +1,121 @@
-require 'CSV'
-require './lib/team'
-require './lib/game'
 
-class StatTracker
-
-  def initialize
-    @teams = []
-    @games = []
-  end
 
-# team_id,franchiseId,teamName,abbreviation,Stadium,link
+class StatTracker
+  attr_reader :game_path, :team_path, :game_teams_path
 
-  def load_from_csv(root_path = "./data")
-    CSV.foreach("#{root_path}/teams.csv", headers: true, header_converters: :symbol) do |row|
-      data = {team_id: row[:team_id],
-              franchiseid: row[:franchiseid],
-              teamname: row[:teamname],
-              abbreviation: row[:abbreviation],
-              stadium: row[:stadium],
-              link: row[:link]
-            }
-      @teams << Team.new(data)
-    end
+  def self.from_csv(file_paths)
+    game_path = file_paths[:games]
+    team_path = file_paths[:teams]
+    game_teams_path = file_paths[:game_teams]
 
-    CSV.foreach("#{root_path}/games.csv", headers: true, header_converters: :symbol) do |row|
-      data = {game_id: row[:game_id],
-              season: row[:season],
-              type: row[:type],
-              date_time: row[:date_time],
-              away_team_id: row[:away_team_id],
-              home_team_id: row[:home_team_id],
-              away_goals: row[:away_goals],
-              home_goals: row[:home_goals],
-              venue: row[:venue],
-              venue_link: row[:venue_link]
-            }
-      @games << Game.new(data)
-    end
+    StatTracker.new(game_path, team_path, game_teams_path)
```

Comment:
Creates out one instance of stat tracker! pulls from all our csvs (two truncated).